apps/alpha-test-app/src/views/HomeView.vue

```diff
@@ -44,6 +45,7 @@ const COMPONENTS = [
   "OnyxSkeleton",
   "OnyxSwitch",
   "OnyxTooltip",
+  "OnyxEmpty",
```

Comment:
alphabetical order pls :) easier to search for what you want to check in the checkbox list then :)